app/globals.css

```
@@ -483,4 +483,31 @@
     outline-offset: 3px;
     box-shadow: none;
   }
+
+  /* Performance: CSS twinkle animation for stars (replaces framer-motion per-star animations) */
+  @keyframes twinkle {
+    0%, 100% { opacity: 0.2; transform: scale(1); }
```

Stylelint (17.2.0)

[error] Expected no more than 1 declaration (declaration-block-single-line-max-declarations)

(declaration-block-single-line-max-declarations)